bugfix.txt

```diff
@@ -0,0 +1,2 @@
+Hi I am bug fix 
```

Comment:
Hi @shan5a6 , why can't we test this with some testing tool & then asses the result.

Reply:
Yes we can do it and I implmeneted it.